### PR TITLE
switched to adaptive icons.

### LINKED ISSUE #67: Add Adaptive Icon
https://developer.android.com/preview/features/adaptive-icons.html
Is related to #46

### PR DESCRIPTION
**Pull request by sakchhams**
resolves #67

### CODE REVIEW

**Comment by abdulwd** [comment]
@sakchhams Your commit history includes adding adaptive icon then reverting and then again adding. So to have a clean commit history you must wrap up your work in a single commit. You can **squash** your commits for doing so.

**Comment by sakchhams** [comment]
@abdulwd is it okay now?

**Comment by abdulwd** [comment]
@sakchhams Still three commits are present. If maintainers don't have a problem with that it can be merged. But to have clean commit history we must complete a small issue in a single commit. You can refer to this [link](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) to learn how to squash your commits.

**Comment by sakchhams** [comment]
@abdulwd It's finally done.

File: app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.java [hunk 1]
```diff
@@ -201,7 +201,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     updateMenu(mViewPager.getCurrentItem());
     toolbar.setOnClickListener(v -> {
       if (mViewPager.getCurrentItem() == 1)
-        MenuItemCompat.expandActionView(menu.findItem(R.id.action_search));
```
Comment (mhutti1) [inline app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.java]: Please only include your changes.

**Comment by sakchhams** [comment]
@mhutti1 I'll resync the repo to a new branch and open a fresh PR

**Comment by sakchhams** [comment]
@mhutti1 cleaned up everything now only includes changes from me.

**Comment by sakchhams** [comment]
@mhutti1 cleaned up everything. Now only includes commits from me. A relative github newbie, pardon the chaos.

**Pull request merged**